custom_item_mod.sln

```diff
@@ -0,0 +1,25 @@
+﻿
```

Comment:
Do we need this file? I always just open the .csproj file. I don't use VS though.

Reply:
It's a good idea to have it for cross compatibility with visual studio - it's commonly used at this point.

Technically it can be ignored if preferred, but including it means that it "just works" for visual studio users, while other editors can continue to load the csproj file